package.json

```diff
@@ -133,42 +141,27 @@
         "type": "string"
       },
       "default": [
-        "**/*.aux",
-        "**/*.aux.bak",
-        "**/*.bbl",
-        "**/*.bcf",
-        "**/*.blg",
-        "**/*.dvi",
-        "**/*.fdb_latexmk",
-        "**/*.fls",
-        "**/*.idx",
-        "**/*.idx.bak",
-        "**/*.ilg",
-        "**/*.ind",
-        "**/*.lof",
-        "**/*.log",
-        "**/*.lol",
-        "**/*.lot",
-        "**/*.nav",
-        "**/*.out",
-        "**/*.pdf",
-        "**/*.ps",
-        "**/*.snm",
-        "**/*.synctex.gz",
-        "**/*.toc",
-        "/**/_minted-{jobname}",
-        "/{output_dir}/sage-plots-for-{jobname}.tex",
+        "$OUTDIR/$JOB!($OUTEXT|.synctex.gz|.tex)",
```

Comment:
Dicy has separate clean levels (shallow/deep) so this was changed to make that work. I'll detail more below.